### ISSUE TITLE
Test Generators Warning

### ISSUE THREAD

**Issue by btecu** [ember-template-lint/ember-cli-template-lint#289]
Getting the following warning in a project that doesn't have a tests suite. Could `ember-cli-template-lint` ignore the test generators when a framework is not detected or specified?

```
WARNING: [ember-cli-template-lint] Test framework detection was unsuccessful. Please provide a "testGenerator" option explicitly to enable the test generators.
```

**Comment by rwjblue** [comment]
Hmm. Doesn’t the suggestion in the warning explain how to fix the issue?

**Comment by rwjblue** [comment]
I don’t fully grok the benefit if you don’t enforce linting via CI....

**Comment by btecu** [comment]
Not sure if I follow, pass `testGenerator: ''` to `ember-cli-template-lint`?

Since there are no tests, what should the `testGenerator` be?

**Comment by Turbo87** [comment]
@btecu the purpose of `ember-cli-template-lint` is to generate tests for the output of `ember-template-lint`. if you don't want to generate tests then you shouldn't use `ember-cli-template-lint`. the test frameworks are detected from the dependencies in your `package.json`. if you don't have either `ember-cli-qunit` or `ember-cli-mocha` in there you will see this warning. without any of those dependencies `ember-cli-template-lint` won't work properly.
